src/CoinParamUtils.cpp

```diff
@@ -839,11 +953,8 @@ void printHelp(CoinParamVec &paramVec, int firstParam, int lastParam,
   if (noHelp) {
     int lineLen = 0;
     for (i = firstParam; i <= lastParam; i++) {
-      CoinParam *param = paramVec[i];
-      if (param == 0)
-        continue;
-      if (param->display() || hidden) {
-        std::string nme = param->matchName();
+      if (paramVec[i]->getDisplayPriority() || hidden) {
```

Comment:
Now the comment in line 941 is lying - if paramVec contains NULL, this line will dereference it.
Are params now supposed to be != NULL?

Reply:
The way the parameter mechanisms of Clp and Cbc are designed, there shouldn't be any NULLs because parameters are indexed by an enum that starts at 0 and has no gaps. But it doesn't hurt to allow for this case, so I'll switch it back to the way it was previously (with the check for NULL). I'm not sure why I changed it to begin with, but I did initially change CoinParamVec to a vector of CoinParam objects, since every element needs to be allocated only at the beginning and deallocated only at the end and this seemed convenient. But I eventually went back to dynamically allocating the each CoinParam object for reasons I've forgotten now.